client/routes.js

```diff
@@ -25,7 +25,6 @@ export default (history) =>
       <Route path="users" component={containers.Users}>
         <Route path=":id" component={containers.User} />
       </Route>
-      <Route path="import-export" component={containers.ImportExport} />
```

Comment:
we should still have a route to import/export inside configuration. We will need it here https://github.com/auth0/auth0-authorization-extension/blob/dev/client/actions/configuration.js#L30
and this line should be changed to '/configuration/import-export'